client/my-sites/plans/ecommerce-trial/trial-feature-card/style.scss

```diff
@@ -13,87 +14,108 @@
 		border-radius: 0 0 2px 2px;
 	}
 
-	.trial-feature-card__illustration {
-		width: 115px;
-		height: 65px;
-
-		@media (max-width: $break-mobile) {
-			display: none;
-		}
-	}
-
-	.trial-feature-card__content {
-		flex: 1;
+	.trial-feature-card__header {
 		display: flex;
-		flex-direction: column;
-		padding: 5px 25px;
+		padding: 24px;
+		cursor: pointer;
+		text-align: left;
+		line-height: 1.5;
 
 		@media (max-width: $break-mobile) {
 			padding: 0;
 		}
 
+		.trial-feature-card__illustration {
+			width: 115px;
+			height: 65px;
 
-		.trial-feature-card__content-title {
-			font-size: $font-title-small;
-			font-weight: 500;
+			@media (max-width: $break-mobile) {
+				display: none;
+			}
 		}
 
-		.trial-feature-card__content-subtitle {
-			font-size: $font-body;
-			color: var(--color-neutral-40);
+		.trial-feature-card__header-text {
+			flex: 1;
+			padding: 0 24px;
 
 			@media (max-width: $break-mobile) {
-				display: none;
+				padding: 0;
+			}
+
+			.trial-feature-card__title {
+				font-size: $font-title-small;
+				font-weight: 500;
+			}
+
+			.trial-feature-card__subtitle {
+				font-size: $font-body;
+				color: var(--color-neutral-40);
+				font-weight: 400;
+
+				@media (max-width: $break-mobile) {
+					display: none;
+				}
 			}
 		}
 
-		.trial-feature-card__content-items {
-			margin-top: 45px;
+		.trial-feature-card__accordion-toggle {
+			cursor: pointer;
+			margin-top: 25px;
 
 			@media (max-width: $break-mobile) {
-				margin-top: 30px;
+				margin-top: 10px;
 			}
 
-			.trial-feature-card__content-items-item {
-				display: flex;
-				padding: 7px 0;
+			.gridicons-chevron-down,
+			.gridicons-chevron-up {
+				fill: var(--color-neutral-90);
+			}
+		}
+	}
 
-				.trial-feature-card__content-items-item-icon {
-					padding: 3px 0;
+	.trial-feature-card__items {
+		flex: 1;
+		display: flex;
+		flex-direction: column;
+		max-width: 590px;
```

Comment:
This rule feels wrong - I think we should be curbing the size of the container, not some specific inner element. If we keep this, the text in the items ends well before the right edge of the card on wide screens, and that looks odd. Things look fine with it removed, both in mobile and in desktop.
```suggestion
```

For reference, why do the lines end/wrap in the middle of the screen?

<img width="1056" alt="Screenshot 2023-02-09 at 22 33 03" src="https://user-images.githubusercontent.com/3376401/217931955-6147200f-d617-494f-867a-f8613bb0f343.png">

Reply:
Figma:
![image](https://user-images.githubusercontent.com/3801502/217935242-ec39cd7a-1cc2-421c-91cf-5fe7f2dfb220.png)

Reply:
Not sure why, though

Reply:
🤷 Then let's go with it for now. We should probably do a walkthrough with the designers early next week.